tests/lib/c_lib/src/main.c

```diff
@@ -183,7 +186,8 @@ void test_strncmp(void)
 
 	/* Note we don't want to count the final \0 that sizeof will */
 	__ASSERT_NO_MSG(sizeof(pattern) - 1 > BUFSIZE);
-	memcpy(buffer, pattern, BUFSIZE);
+	zassert_not_null(memcpy(buffer, pattern, BUFSIZE),
+		     "memcpy return null");
```

Comment:
No need to add this. The purpose of `test_strncmp` is not to verify `memcpy`.

Reply:
OK, I have fixed it.